platforms/hyperledger-fabric/configuration/roles/create/anchorpeer/tasks/nested_anchorpeer.yaml

```diff
@@ -32,6 +32,8 @@
     git_branch: "{{ org.gitops.branch }}"
     charts_dir: "{{ org.gitops.chart_source }}"
     vault: "{{ org.vault }}"
+    fabrictools_image: "hyperledger/fabric-tools:{{ fabric_version }}"
```

Comment:
directly use network.version  and network.docker.url here, just to reduce the number of file changes.